parquet-hadoop/README.md

```diff
@@ -158,7 +164,7 @@ This property should be between 0 and 1.
 ---
 
 **Property:** `parquet.page.size.row.check.max`  
-**Description:** The maximum number of rows per page.  
+**Description:** The frequency of checks of the page size limit. In other words, we perform the checking after each `parquet.page.size.row.check.max` rows.  
```

Comment:
It is not that simple and `parquet.page.size.row.check.min` shall be updated as well.
So, the frequency of the checks will be between `parquet.page.size.row.check.min` and `parquet.page.size.row.check.max`. You may influence the checking frequency by setting these numbers. (Higher frequency results more precise page sizes, lower frequency improves performance.) See the [code](https://github.com/apache/parquet-mr/blob/master/parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnWriteStoreBase.java#L221-L266) for more details.

Reply:
Thanks @gszadovszky for the clarification, I'll duplicate the description for both `parquet.page.size.row.check.min` and `parquet.page.size.row.check.max` properties.